portfolio/src/main/java/com/google/sps/servlets/ReturnRandomNumberServlet.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.servlets;
+
+
+public class ReturnRandomNumberServlet {
+
+    protected final int Max_value = 3;
```

Comment:
```suggestion
    private static final int maxValue = 3;
```

Ditto for the other one.